generate_protos.sh

```diff
@@ -4,6 +4,10 @@ DIR=`pwd`
 rm -rf $DIR/gen
 LYFT_IMAGE="lyft/protocgenerator:8167e11d3b3439373c2f033080a4b550078884a2"
 SWAGGER_CLI_IMAGE="docker.io/lyft/swagger-codegen-cli:dc5ce6ec6d7d4d980fa882d6bd13a83cba3be3c3"
+PROTOC_GEN_DOC_IMAGE="pseudomuto/protoc-gen-doc:1.4.1"
```

Comment:
I know this is a kinda tedious ask, but could we check in the Dockerfile and maybe even add automation (can also be filed as a todo/issue that someone else on the team can take up) so that we can easily update this image if need be?

Reply:
oh I didn't realize this was the official image, nevermind - please disregard this comment :-)

Reply:
Yeah, using the official image for now since I figured that'd be easiest. We _could_ set up a build using the [fork](https://github.com/flyteorg/protoc-gen-doc), but I'm not sure that's worth for what seems to be needed/used at the moment.